src/components/Player/Pages/Histograms/Histograms.jsx

```diff
@@ -7,17 +7,20 @@ import { HistogramGraph } from 'components/Visualizations';
 import ButtonGarden from 'components/ButtonGarden';
 import histogramNames from 'components/Player/Pages/matchDataColumns';
 import { TableFilterForm } from 'components/Form';
+import { ContentContainer } from 'components/ContentContainer';
 import { browserHistory } from 'react-router';
 
-const Histogram = ({ routeParams, columns, playerId }) => (
+const Histogram = ({ routeParams, columns, playerId, error, loading }) => (
   <div style={{ fontSize: 10 }}>
     <TableFilterForm />
     <ButtonGarden
       onClick={buttonName => browserHistory.push(`/players/${playerId}/histograms/${buttonName}${window.location.search}`)}
       buttonNames={histogramNames}
       selectedButton={routeParams.subInfo || histogramNames[0]}
     />
-    <HistogramGraph columns={columns || []} />
+    <ContentContainer error={error} loading={loading}>
```

Comment:
Could we just use ContentContainer for tables instead of a separate TableContainer?